main.py

```diff
@@ -321,16 +119,16 @@ def collision(self):
             player_life -= 1
             enemy_list.remove(i)
             if player_life == 0:
-                draw.text(((width - textsize_w_over)/2, (height - textsize_h_over)/2), text['game_over'], (255,0,0),font=fnt3)
-                draw.text(((width - textsize_w_score)/2, (height - textsize_h_over)/2+40), text['score'], font=fnt)
-                draw.text(((width - textsize_w_score)/2+textsize_w_score, (height - textsize_h_over)/2+40), str(score),font=fnt)
+                draw.text(((WIDTH - textsize_w_over)/2, (HEIGHT - textsize_h_over)/2), text['game_over'], (255,0,0),font=FNT3)
```

Comment:
```suggestion
                text_width = (WIDTH - text_game_over_width) / 2
                text_height = (HEIGHT - text_game_over_height) / 2
                text_position = (text_width, text_height)
                draw.text(
                  text_position, 
                  text['game_over'], 
                  (255,0,0), 
                  font=FNT3,
                )
```
이런식으로 변수명도 의미가 조금 더 명확하게 수정하고 코드 정리 필요